src/routes/moderator/(useCardKey)/+layout.svelte

```diff
@@ -0,0 +1,11 @@
+<script lang="ts">
+  import { cardKeyScanStore } from '$lib/utils/cardKeyScanStore';
+  import { onDestroy } from 'svelte';
+
+  const unsubscribe = cardKeyScanStore.subscribe(() => {});
+  onDestroy(() => {
+    unsubscribe();
+  });
```

Comment:
See Mayb-confusing-svelte-things :P. We subscribe to it to ensure it does not deleted, otherwise the user would have to re-register their iodevice whenever they navigated between the `/moderator`-endpoints